### DISCUSSION TITLE
Boost for Sonic

### DISCUSSION THREAD

**Discussion by UltramanTime** [Techokami/SonicWorldsNext#59] [Q&A] [answered]
Another Question:
Will Sonic have Boost?

Source: [https://sonic.fandom.com/wiki/Boost](url)

**Comment by Techokami** [comment, marked as answer]
No. IF we do anything Modern-era, it will be like Worlds Delta with a separate character, but said character will NOT be part of the actual codebase; it would instead be a series of written tutorials so you can do it yourself.
